docs/content/concepts/configuration/config-schema-legacy.mdx

```diff
@@ -0,0 +1,296 @@
+
+Below we define a simple op and asset with identical `config_schemas` defining a single configurable parameter, `person_name`, as well as a resource with a configurable `url` parameter:
+
+```python file=/concepts/configuration/configurable_op_asset_resource.py startafter=start endbefore=end
+@op(config_schema={"person_name": str})
+def op_using_config(context):
+    return f'hello {context.op_config["person_name"]}'
+
+
+@asset(config_schema={"person_name": str})
+def asset_using_config(context):
+    # Note how asset config is also accessed with context.op_config
+    return f'hello {context.op_config["person_name"]}'
+
+
+@resource(config_schema={"url": str})
+def resource_using_config(context):
+    return MyDatabaseConnection(context.resource_config["url"])
+```
+
+<Note>
+  It is technically possible to access <code>context.op_config</code> inside ops
+  (not assets) without defining a <code>config_schema</code>. However, this is
+  not recommended.
+</Note>
+
+You can also build config into jobs, as described in [the Jobs documentation](/concepts/ops-jobs-graphs/jobs#configuring-jobs).
+
+---
+
+## Specifying runtime configuration
+
+If you want to execute `op_using_config` or materialize `asset_using_config`, we'll need to provide values for the parameters specified in `config_schema`. How we provide these values depends on the interface we are using:
+
+- [Python](#python)
+- [Dagit](#dagit)
+- [Command line](#command-line)
+
+### Python
+
+From the Python API, we can use the `run_config` argument for <PyObject object="JobDefinition" method="execute_in_process"/> or <PyObject object="materialize"/>. This takes a dictionary where configuration values for ops/assets are specified under `ops.<op_or_asset_name>.config` (for resources under `resources.<resource_name>.config`):
+
+```python file=/concepts/configuration/execute_with_config.py startafter=start_execute_with_config endbefore=end_execute_with_config dedent=4
+@job
+def example_job():
+    op_using_config()
+
+job_result = example_job.execute_in_process(
+    run_config={"ops": {"op_using_config": {"config": {"person_name": "Alice"}}}}
+)
+
+asset_result = materialize(
+    [asset_using_config],
+    run_config={
+        "ops": {"asset_using_config": {"config": {"person_name": "Alice"}}}
+    },
+)
+```
+
+### Dagit
+
+From Dagit's [Launchpad](/concepts/dagit/dagit#launchpad-tab), we supply config as YAML using the config editor. The editor has typeahead, schema validation, and schema documentation. You can also click the "Scaffold Missing Config" button to generate dummy values based on the config schema. Note that a modal containing the launchpad editor will pop up if we attempt to materialize an asset with a defined `config_schema`:
+
+<Image
+alt="Config in Dagit"
+src="/images/concepts/config-dagit.png"
+width={3808}
+height={2414}
+/>
+
```

Comment:
This speaks to the point of having a language to describe the launchpad in the context of the new system. This is not "legacy" now. The Launchpad still works as described.

Reply:
This is a good point, admittedly one I hadn't thought of in my initial review.

Perhaps the way to approach this is to point users to these same sections in the new config guide? Something like `"Refer to [these instructions](/link/to/new/config/guide) for using/accessing runtime config in the Dagster UI"`

It would mean we don't have to maintain the same content in multiple places, and this is a fairly standard practice for SaaS product documentation when things are updated/deprecated.